srcpkgs/pam_ssh_agent_auth/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'pam_ssh_agent_auth'
+pkgname=pam_ssh_agent_auth
+version=0.10.3
+revision=1
+archs="i686 x86_64"
+build_style=gnu-configure
+make_build_args="LD=gcc"
```

Comment:
This is most likely wrong and breaks cross compilation, should use `$CC`.

Reply:
The Makefile generated expects it. Not having it would require patching the Makefile or configure script.